README.md

```diff
@@ -1 +1,97 @@
-# yapapi-service-manager
\ No newline at end of file
+# yapapi-service-manager
+
+Helper tool for management of [Golem](https://handbook.golem.network/)-based services.
+
+## yapapi-service-manager vs yapapi
+
+The official Golem python requestor development library is [yapapi](https://github.com/golemfactory/yapapi).
+`yayapi` is used internally in `yapapi-service-manager`, so there's exactly nothing this library can do that is not available in pure `yapapi`.
+
+`yapapi-service-manager` provides a higher-level services API than `yapapi`. Main features:
+
+* create/destroy services on demand
+* persistent wrapper objects that are created before agreement is signed & stay after it was terminated
+* most of the interface is `sync` (although this is still an `async` library that will not work when called in non-async context)
+
+On the other hand, if you need either one of
+
+* batch API (TODO: yayapi docs url)
+* efficient way of spawning multiple services in clusters (TODO: yayapi docs url)
+* stable backward-compatible API & support
+
+than you should use `yapapi`.
+
+Note: this library changes the way services are *managed*, but the way they are *defined* is exactly the same as in `yapapi`.
+
+More general, this library should be considered a temporary stage in requestor API development.
+In the long term, `yapapi-service-manager` will either be merged into `yapapi` (with possible serious API changes) or abandoned.
+
+
+## Examples
+
+1. Simple service that just prints provider time few times. This is pretty useless, just demonstrates the base usage.
+
+
+```
+$ python3 examples/clock.py
+```
+
+2. "Standard" interactive python console, but running on a provider machine.
+   
+
+```
+$ python3 examples/python-shell.py
+```
+
+3.  More complex usage: [Erigon](https://github.com/golemfactory/yagna-service-erigon). Features:
+
+* custom runtime (TODO: runtime docs url)
+* `yapapi-service-manager` integrated with [Quart](https://pgjones.gitlab.io/quart/) http server
+    
+
+## API reference
```

Comment:
I don't understand why.

Is this about the form? Or are there some pieces of information that should be included in the `API reference` and are missing in this section?

(having said that, I'll change it because you're the naming guy ;p)

Reply:
"API reference" is a list of methods/functions/classes included in the API with descriptions. There's nothing wrong with this section, it just shouldn't be called "API reference".